tasks.py

```diff
@@ -0,0 +1,117 @@
+import os
+from pathlib import Path
+from shutil import which
+
+from invoke import task
+from invoke.util import cd
+
+TOOLS = ["poetry", "pre-commit"]
+PORT = os.environ.get("SERVER_PORT", 8000)
+
+ACTIVE_VENV = os.environ.get("VIRTUAL_ENV", None)
+VENV_HOME = Path(os.environ.get("WORKON_HOME", "~/.local/share/virtualenvs"))
+VENV_PATH = Path(ACTIVE_VENV) if ACTIVE_VENV else (VENV_HOME / "kagi")
+VENV = str(VENV_PATH.expanduser())
+
+PRECOMMIT = which("pre-commit") if which("pre-commit") else (VENV / "pre-commit")
+POETRY = which("poetry") if which("poetry") else (VENV / "poetry")
+
+
+@task
+def build(c, docs=True):
```

Comment:
```suggestion
def build(c, docs=True):
    if docs:
        build_docs(c)

def build_docs(c):
```

Reply:
To be honest, I am not sure about the build command at all, it seems a bit too generic.

Reply:
Agreed. As of bad1205 those are now `docs` and `viewdocs` tasks.